docs/next/.versioned_content/_versions.json

```diff
@@ -1,22 +1,5 @@
 [
   "master",
-  "0.11.0",
```

Comment:
Shouldn't our older versions only need to be rebuilt in exceptional circumstances? In theory, we should only need to build docs for the specific release and then merge them in with the rest of our versioned content.

Reply:
Yea I think that's the ideal case but I'm still figuring out how - I think the end state is to only build the latest and get the older versions from a static source.